src/app/meet/[meetingId]/page.tsx

```
@@ -0,0 +1,640 @@
+					</div>
+				</div>
+
+				{/* Timer + View Toggle + Recording */}
+				<div className="flex items-center gap-2">
+					{/* View Mode Toggle */}
+					<div className="flex items-center bg-white/5 backdrop-blur-md border border-white/10 rounded-full p-0.5">
+						<button
+							className={`flex items-center gap-1 px-2.5 py-1 rounded-full text-xs font-medium transition-all ${viewMode === "speaker" ? "bg-indigo-500/30 text-indigo-300" : "text-gray-500 hover:text-gray-300"}`}
+							onClick={() => setViewMode("speaker")}
+							title="Speaker view"
+						>
```

Biome (2.4.14)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)